### PR TITLE
Editorial: Information Access, fixes #304

### LINKED ISSUE #304: Copy edit section 2.3, Information Access

### PR DESCRIPTION
**Pull request by rhiaro**
Condenses section into a single principle, with explanations. Removes many words.

See https://github.com/w3ctag/privacy-principles/pull/321 for edits in context.

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/325.html" title="Last updated on Sep 6, 2023, 4:29 PM UTC (fed1cbb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/325/077d2a9...fed1cbb.html" title="Last updated on Sep 6, 2023, 4:29 PM UTC (fed1cbb)">Diff</a>

### CODE REVIEW

**Comment by rhiaro** [comment]
@jyasskin I realise there might be a specific browser-engine-y reason this was three distinct principles, so apologies if I've lost something important by trying to turn them into one! It did read a bit like it was trying to address something/someone very specific, and in doing so obfuscated the "key message", which is what I've tried to re-surface with the edits.

File: index.html
Comment (jyasskin) [inline index.html:1403]: It's not quite that the APIs shouldn't be easier to use, but that they shouldn't be easier to use to get access to the bad information. Can we just remove the "nor should" part, or does that lose something useful?

```suggestion
not make that information easier to access compared to existing comparable
web platform features.
```

File: index.html
Comment (jyasskin) [inline index.html:1405]: This is just me being pedantic, so feel free to ignore if you don't want to add the words:

```suggestion
Specifications describing these APIs should also:
```

File: index.html
Comment (jyasskin) [inline index.html:1411]: It's actually for a user agent that blocks specifically the information that the new API would redundantly expose. Removing "in general" here might be enough, although we still have "this kind of information" which is broader than ideal.

```suggestion
  information (perhaps by breaking some experiences on the web that
```

Another approach might be around rephrasing the paragraph to talk about "A specification for a new API that redundantly exposes some undesirable information should: ... make it clear how any [=user agent=] whichblocks access to this information (perhaps...".

File: index.html
Comment (jyasskin) [inline index.html:1417]: Trying to make the web-without-3p-cookies reference flow better:

```suggestion
  Usually, these APIs will be designed to expose data that enables some
  [=appropriate=] information discovery, as recommended by [[[web-without-3p-cookies]]]:
  
  > It is better to approach [these use cases] with replacement technologies
  that are designed-for-purpose and built to respect user privacy.
```

**Review by jyasskin — APPROVED** [review]
I think this is good, once we've considered several comments. @yoavweiss, any concerns?

File: index.html
Comment (jyasskin) [inline index.html:1348]: Oh, right, I wanted to think about the principle itself after I read the whole new section. I originally wrote 3 principles because we'd figured out that there were 3 kinds of new APIs that got proposed, and each one implied a different set of constraints. You've covered the constraints in the section, and I'm not super concerned about whether we describe the constraints as principles or just in prose. The remaining question is how to summarize the point of the overall section, and I'm not sure this is actually the right principle for that...

Maybe it's more like:

```suggestion
  <span class="practicelab">New Web APIs must guard users' information at least
  as well as existing APIs that are expected to stay in the web platform.</span>
```

Or something about the "target state of the web platform", but "target state" might need too much explaining.

Comment (jyasskin) [reply index.html:1348]: Or "at least as well as our concrete goals for the web platform"?

File: index.html
Comment (yoavweiss) [inline index.html:1377]: Maybe a variant of "unless it is possible to include the new APIs as part of the same plan, and adding these new APIs won't make the plan harder"?

I'd also s/plan/plausible plan/

File: index.html
Comment (yoavweiss) [inline index.html:1401]: I fundementally disagree with this section - if there's information with both bad and good uses, creating easier APIs to access it would hurt the good uses of that information while not necessarily impacting the bad uses (as they are typically more lucrative and deployed by folks that care less about the user experience).  E.g. based on this and the asides above, we wouldn't ship `ping` and IntersectionObserver. I don't think there's consensus that we regret shipping neither.

So a couple of caveats I'd like to see here:
* There needs to be consensus that the information is something we'd like to unship from the platform.
* The new APIs make it significantly cheaper (by an order of magnitude) to access that information.

File: index.html
Comment (chrisn) [reply index.html:1384]: I think we should avoid the word "sketchy" (even as an anchor), it could be misunderstood by people where English is a second language, and in English it has multiple meanings - lacking detail / being misleading, neither of which really captures what's being described here.

File: index.html
Comment (rhiaro) [reply index.html:1357]: I was unsure about swapping in `appropriate`, but I felt "acceptable" begs the question of: to whom? I think if we're going to use "acceptable" this way we should define it somewhere.

File: index.html
Comment (rhiaro) [reply index.html:1405]: Fewer words always better!

(Held lightly; if you think it adds clarity, more words is better. I'll leave to you.)

File: index.html
Comment (rhiaro) [reply index.html:1348]: I think the suggested edit is clear. I wouldn't want to complicate it by referring to "target state" or "concrete goals". This might want more noodling on by the group though, so I won't add the commit right now.

File: index.html
Comment (rhiaro) [reply index.html:1377]: I think I understand the sense of "plausible" here, but it's potentially ambiguous/subjective. Perhaps something that leans on the standards process? A "concrete, consensus-based plan"? A "plan supported by multiple stakeholders on the web pltaform"?

File: index.html
Comment (rhiaro) [inline index.html:1385]:
```suggestion
that can be used to undermine people's privacy. It
```

File: index.html
Comment (rhiaro) [inline index.html:1384]:
```suggestion
### Unavoidable information exposure

Some functionality of the web has historically been provided using features
```

Comment (rhiaro) [reply index.html:1384]: I've suggested "Unavoidable information exposure" instead

**Comment by rhiaro** [comment]
I've added some of the suggestions, left a few comments, and there are a few things to be resolved still (particularly @yoavweiss's comments). If the group discusses this today, I can do some more edits based on consensus later. Or, if it's easier for someone else to do the edits so it can be merged today, go ahead without me.

File: index.html
Comment (jyasskin) [inline index.html:1356]:
```suggestion
access</dfn> to data on the web. Whether or not web APIs should expose information
depends on the suitability of these [=access guards=] in
```

File: index.html
Comment (npdoty) [reply index.html:1407]: +1 to the clarity that these are additions to specifications, not APIs.

File: index.html
Comment (jyasskin) [inline index.html:1357]:
```suggestion
access</dfn> to data on the web. When designing Web APIs, use access guards
to ensure they expose information in [=appropriate=] ways.
```

File: index.html
Comment (hober) [inline index.html:1361]: Might be worth pointing out that even `` is also gated on JavaScript being enabled?

**Comment by npdoty** [comment]
Condensed principle here might cover all three cases, but does the explanatory text cover all the different cases sufficiently?

File: index.html
Comment (jyasskin) [reply index.html:1361]: `` is a good example of us following the overall principle in this section, but it doesn't really fit into this paragraph. Maybe a second paragraph:

> When the <{img/loading}>=<{img/loading/lazy}> attribute was added, the designers realized that it exposed the scroll position, so it's also guarded by the setting to turn off JavaScript.

I'm not sure those [linking shorthands](https://speced.github.io/bikeshed/#autolink-wrappers) are supported by Respec...

File: index.html
Comment (jyasskin) [inline index.html:1361]:
```suggestion
  For example, the URLs of resources, the timing of link clicks, and the referrer chain within a
  single origin are not guarded by anything; the scroll position is guarded by the setting to turn off
  JavaScript; and access to the camera or geolocation are guarded by permission prompts.
  
  When the `<img loading=lazy>` attribute was added, the designers realized
  that it exposed the scroll position, so it's also guarded by the setting to turn
  off JavaScript.
```

File: index.html
Comment (jyasskin) [reply index.html:1406]: FWIW, I think this is a comment about the original text, rather than this change, and we should sit down once this PR is merged and try to fix up the whole section to match your understanding better. I don't think it should block merging this editorial PR.

**Pull request merged**
